lib/Dialect/TritonGPU/Transforms/Pipeliner/MatmulLoopPipeline.cpp

```diff
@@ -51,14 +51,107 @@ struct LoadInfo {
 
 } // namespace
 
-static void createAsyncCopy(scf::ForOp &forOp, tt::LoadOp loadOp, Value alloc,
-                            Value insertIdx, Value extractIdx,
-                            tt::CoarseSchedule &schedule,
-                            tt::CoarseSchedule::Cluster prefetchCluster,
-                            llvm::MapVector<Operation *, LoadInfo> &loadToInfo,
-                            int numStages) {
-  OpBuilder builder(forOp);
-  Value zero = builder.create<arith::ConstantIntOp>(forOp.getLoc(), 0, 32);
+class OpBuilderWrapper : public OpBuilder {
+public:
+  explicit OpBuilderWrapper(Operation *op,
+                            OpBuilder::Listener *listener = nullptr)
+      : OpBuilder(op, listener) {}
+  explicit OpBuilderWrapper(Region &region, Listener *listener = nullptr)
+      : OpBuilder(region, listener) {}
+
+  template <typename OpTy, typename... Args>
+  OpTy createWrapper(Location location, int stage, int cluster,
```

Comment:
Nit: createWithStage or createWithPipelineInfo

Reply:
Updated in the latest diff.